src/inputs/Table/Table.js

```diff
@@ -412,6 +503,41 @@ Table.propTypes = {
    * If not defined, this button will not be rendered.
    */
   onExport: PropTypes.func,
+  /*
+   * Callback function that will be called when user click on the add row button.
+   * If not defined, this button will not be rendered.
+   */
+  onAddRow: PropTypes.func,
+  /*
+   * Callback function that will be called when user click on the delete row button.
+   * If not defined, this button will not be rendered.
+   */
+  onDeleteRow: PropTypes.func,
+  /*
+   * Function to update the label of the delete confirmation dialog body
+   */
+  buildDialogBodyLabel: PropTypes.func.isRequired,
+};
+
+const DEFAULT_LABELS = {
+  clearErrors: 'Clear',
+  label: 'undefinedTable',
+  loading: 'Loading...',
+  errorsPanelMainError: 'File load failed.',
+  placeholderTitle: 'Import your first data',
+  placeholderBody: 'After importing a valid csv or xlsx file, your data will be displayed in an interactive table.',
+  import: 'Import',
+  export: 'Export',
+  addRow: 'Add a new row',
+  deleteRows: 'Remove selected rows',
+  deleteRowsDialog: {
+    title: 'Delete selected lines?',
+    getBody: () => 'Are you sure you want to delete these lines?',
+    cancel: 'Cancel',
+    confirm: 'Delete',
+    checkbox: "Don't show this message again",
+  },
+  fullscreen: 'fullscreen',
```

Comment:
If this label is used as a tooltip, it should probably start with a capital letter

Reply:
DONE